pkg/sql/schema_changer.go

```diff
@@ -1095,9 +1189,14 @@ func (sc *SchemaChanger) done(ctx context.Context) error {
 				if isRollback {
 					description = "ROLLBACK of " + description
 				}
-
-				if err := sc.createIndexGCJob(ctx, idx.GetID(), txn, description); err != nil {
-					return err
+				if isTemporaryIndex(idx) {
+					if err := sc.createTemporaryIndexGCJob(ctx, idx.GetID(), txn, "TEMPORARY INDEXES USED DURING INDEX BACKFILL"); err != nil {
```

Comment:
Is the all caps to style? fine with it if it is, just checking

Reply:
Thanks for catching this, I meant to fix it up, I made it all caps to make it easier to spot in assertion failures in some of the logic tests and then didn't change it back.